packages/twenty-front/src/modules/object-record/record-picker/multiple-record-picker/hooks/useMultipleRecordPickerPerformSearch.ts

```diff
@@ -77,28 +100,86 @@ export const useMultipleRecordPickerPerformSearch = () => {
           pickedRecordIds: selectedPickableMorphItems.map(
             ({ recordId }) => recordId,
           ),
+          offset: loadMore ? paginationState.currentOffset : 0,
+          limitPerObject: MULTIPLE_RECORD_PICKER_PAGE_SIZE,
         });
 
-        const pickedMorphItems = pickableMorphItems.filter(
-          ({ isSelected }) => isSelected,
+        const existingMorphItems = loadMore
```

Comment:
I'm unsure why the fact that we are in "loadMore" case changes the way we compute results. It should not matter IMO